### PR TITLE
Superweapon Safe and spawner

### PR DESCRIPTION
**Pull request by Stop-Signs**
## About the PR
adds a superweapon safe and groundloot spawner for mappers. it can currently either be the beam devastator or the RPG.
(also fixes a dumb typo I previously made cause im a silly goober)

## Why / Balance
I asked @IamVelcroboy  about mapping the devastator and he suggested I make this

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml:1]: shouldnt exist, you lose a rocket

File: Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/security.yml
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/security.yml:88]: for all of these amount: 1 is the default and can be removed

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/security.yml
Comment (IamVelcroboy) [reply Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/security.yml:88]: Use a table and give it 20% probability for extra rocket.

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml
Comment (IamVelcroboy) [reply Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml:1]: Use a table and give it 20% probability for extra rocket.

File: Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/security.yml
Comment (IamVelcroboy) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/security.yml:91]: `prob: 0.20`

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml:3]: this still shouldnt exist when the [Superweapon] one can have an extra rocket

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml]: unindent these by 2 spaces

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml]: unindent

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml
Comment (IamVelcroboy) [reply Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml:3]: That's why I requested they change that as well to a prob: .20 so they are the same

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml
Comment (IamVelcroboy) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml:63]: This is wrong. I pinged you on Discord with the proper table setup

**Comment by Stop-Signs** [comment]
OK so, after some recommendations from velcro and me figuring out how to make the tables work, it now works as intended. 

the probability was messing with the chance for the RPG so ill let direction decide whether they get it or not.

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml]: why does this need to exist

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml]: can't you just do a groupselector with `GunSafeRocketLauncher` and `GunSafeBeamDevastator` as children like above?

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml]: type > parent > id > suffix > components

**Review by MilonPL — DISMISSED** [review]

**Comment by Stop-Signs** [comment]
Just to make it clear, I asked direction (lady) about the spare rocket and she said it was fine
image

**Review by IamVelcroboy — APPROVED** [review]

**Comment by IamVelcroboy** [comment]
@MilonPL @deltanedas if you are satisfied and can dismiss your change requests, this can be merged

**Review by deltanedas — APPROVED** [review]

**Comment by deltanedas** [comment]
20% can be removed later

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml:66]: EOF newline

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml:21]: newline

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml:5]: parent after type

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Comment by MilonPL** [comment]
fuck!!

**Comment by deltanedas** [comment]
can be changed later, at least it can get mapped now

**Pull request merged**
